src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,69 @@
+package core.basesyntax;
+
+import core.basesyntax.filerider.CsvFileReaderServiceImpl;
+import core.basesyntax.filewriter.FileWriterService;
+import core.basesyntax.operationhandler.BalanceHandler;
+import core.basesyntax.operationhandler.OperationHandler;
+import core.basesyntax.operationhandler.PurchaseHandler;
+import core.basesyntax.operationhandler.ReturnHandler;
+import core.basesyntax.operationhandler.SupplyHandler;
+import core.basesyntax.reportcreator.ReportCreator;
+import core.basesyntax.transactionparser.CsvTransactionParser;
+import java.util.List;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        Storage storage = new Storage();
+
+        final OperationHandler purchaseHandler = new PurchaseHandler();
+
+        final OperationHandler returnHandler = new ReturnHandler();
+
+        final OperationHandler supplyHandler = new SupplyHandler();
+
+        final OperationHandler balanceHandler = new BalanceHandler();
+
+        supplyHandler.handleOperation(new FruitTransaction(Operation
+                .SUPPLY,"Orange", 20), storage);
+        supplyHandler.handleOperation(new FruitTransaction(Operation
+                .SUPPLY,"Orange", 20), storage);
+        supplyHandler.handleOperation(new FruitTransaction(Operation
+                .SUPPLY,"Orange", 20), storage);
+        supplyHandler.handleOperation(new FruitTransaction(Operation
+                .SUPPLY,"Apple", 20), storage);
+        supplyHandler.handleOperation(new FruitTransaction(Operation
+                .SUPPLY,"Apple", 20), storage);
+        supplyHandler.handleOperation(new FruitTransaction(Operation
+                .SUPPLY,"Pea", 20), storage);
+        supplyHandler.handleOperation(new FruitTransaction(Operation
+                .SUPPLY,"Pea", 20), storage);
+        purchaseHandler.handleOperation(new FruitTransaction(Operation
+                .PURCHASE,"Apple", 10), storage);
+        returnHandler.handleOperation(new FruitTransaction(Operation
+                .RETURN,"Apple", 5), storage);
+        returnHandler.handleOperation(new FruitTransaction(Operation
+                .RETURN,"Apple", 10), storage);
+        supplyHandler.handleOperation(new FruitTransaction(Operation
+                .SUPPLY, "Kiwi", 20), storage);
+
+        balanceHandler.handleOperation(null, storage);
```

Comment:
Cannot get idea why this code is here.
You should start processing fro file reading, not some predefined operations
```suggestion
```